documentation/design-docs/ipc-protocol.md

```diff
@@ -669,42 +704,341 @@ Payload
 ```
 Followed by an Optional Continuation of a `nettrace` format stream of events.
 
-### `StopTracing`
+### `CollectTracing5`
 
-Command Code: `0x0201`
+Command Code: `0x0206`
 
-The `StopTracing` command is used to stop a specific streaming session.  Clients are expected to use this command to stop streaming sessions started with [`CollectStreaming`](#CollectStreaming).
+The `CollectTracing5` command is an extension of the `CollectTracing4` command. It has all the capabilities of `CollectTracing4` and introduces new fields to enable a Linux-only user_events-based eventpipe session and to prescribe an allow/deny list for Event IDs. When the user_events-based eventpipe session is enabled, the file descriptor and SCM_RIGHTS of the `user_events_data` file must be sent through the optional continuation stream as [described](#passing_file_descriptor). The runtime will register tracepoints based on the provider configurations passed in, and runtime events will be written directly to the `user_events_data` file descriptor. The allow/deny list of Event IDs will apply after the keyword/level filter to determine whether or not that provider's event will be written. When using this command, even without leveraging the new user_events-based eventpipe session option, the new fields must be serialized.
+
+> Note available for .NET 10.0 and later.
 
 #### Inputs:
 
-Header: `{ Magic; 28; 0x0201; 0x0000 }`
+Header: `{ Magic; 20 + Payload Size; 0x0206; 0x0000 }`
 
-* `ulong sessionId`: The ID for the streaming session to stop
+#### [Streaming Session](#streaming-session) Payload:
+* `uint output_format`: 0
+* `uint circularBufferMB`: The size of the circular buffer used for buffering event data
+* `uint format`: 0 for the legacy NetPerf format and 1 for the NetTrace V4 format
+* `ulong rundownKeyword`: Indicates the keyword for the rundown provider
+* `bool requestStackwalk`: Indicates whether stacktrace information should be recorded.
+* `array<provider_config> providers`: The providers to turn on for the session
 
-#### Returns:
+The Streaming Session `provider_config` is composed of the following data:
```

Comment:
Suggestion for clarity: you can define two separate types `user_events_provider_config` and `streaming_provider_config` rather than describing two different definitions of the same type.

Reply:
Thanks, split it into two